src/paper/serializers/paper_serializers.py

```diff
@@ -1037,6 +1062,29 @@ def get_pdf_url(self, paper):
             return paper.pdf_url
         return None
 
+    def get_version(self, paper):
+        try:
+            paper_version = PaperVersion.objects.get(paper=paper)
+        except PaperVersion.DoesNotExist:
+            return None
+
+        return paper_version
+
+    def get_version_list(self, paper) -> list:
```

Comment:
Nit:

```suggestion
    def get_versions(self, paper) -> list:
```

Reply:
The field is called version_list to stay consistent with other list fields.